docs/tasks/run-application/configure-pdb.md

```diff
@@ -88,6 +88,12 @@ of the evicted pod. `minAvailable` can be either an absolute number or a percent
 of the number of pods from that set that can be unavailable after the eviction. 
 It can be either an absolute number or a percentage.
 
+**Note:** Before v1.9, when creating a `PodDisruptionBudget` object using the
+`kubectl` command line tool, the `minAvailable` field is set to 1 as its default
+value if neither `minAvailable` nor `maxAvailable` is specified. This defaulting
+behavior is removed since v1.9.
+{: .note}
+
```

Comment:
I was targeting master branch because the added text is not causing confusion when merged into any release branch.
I'm okay with rebasing. Just curious about what kind of contents are supposed to be based on a specific release branch.

Reply:
@tengqm I requested the change because it's documentation addressing a version that hasn't been released yet. Make sense?

Reply:
Okay, now I get the idea.